docs/content/workflows/design/sidebar.md

```diff
@@ -0,0 +1,25 @@
++++
+title = "Sidebar"
+weight = 8
+
++++
+
+The "select box" to select a git branch or a version on UI has been removed. There is only one select box for filter on CDS Tags.
```

Comment:
is it useful for newcomers to talk about previous select box ? Maybe an example is better.

For example if you want to know on which branch the build was launched you just have to filter on a specific CDS tag (in this case "git.branch")